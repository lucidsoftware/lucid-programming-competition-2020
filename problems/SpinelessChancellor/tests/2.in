3
READ
MORE
BOOK
217
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE yes
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ yes
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK no
READ yes
MORE no
BOOK yes
READ no
MORE yes
BOOK unsure
READ yes
MORE no
BOOK unsure
READ no
MORE yes
BOOK unsure
READ yes
MORE no
BOOK unsure
READ no
MORE yes
BOOK unsure
READ yes
MORE no
BOOK unsure
READ no
MORE yes
BOOK unsure
READ yes
MORE no
BOOK unsure
READ no
MORE yes
BOOK unsure
READ yes
MORE no
BOOK unsure
READ no
MORE yes
BOOK unsure
READ yes
MORE unsure
BOOK unsure
