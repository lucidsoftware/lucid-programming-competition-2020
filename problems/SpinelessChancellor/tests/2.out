READ yes
MORE abstain
BOOK no
